pkg/testutils/mock_config.go

```diff
@@ -92,27 +71,12 @@ func AddDataVolumeAPI(crdInformer cache.SharedIndexInformer) {
 	})
 }
 
-func UpdateFakeClusterConfig(configMapInformer cache.SharedIndexInformer, cfgMap *v1.ConfigMap) {
-	copy := copy(cfgMap)
-	configMapInformer.GetStore().Update(copy)
-}
-
 func UpdateFakeKubeVirtClusterConfig(kubeVirtInformer cache.SharedIndexInformer, kv *KVv1.KubeVirt) {
-	copy := kv.DeepCopy()
-	copy.ResourceVersion = rand.String(10)
-	copy.Name = "kubevirt"
-	copy.Namespace = "kubevirt"
-
-	kubeVirtInformer.GetStore().Update(copy)
-}
+	clone := kv.DeepCopy()
+	clone.ResourceVersion = rand.String(10)
+	clone.Name = "kubevirt"
+	clone.Namespace = "kubevirt"
+	clone.Status.Phase = "Deployed"
```

Comment:
@nunnatsa same here about the initialization `clone.Status.Phase = "Deployed"`

Reply:
@enp0s3 - same answer.